coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -126,19 +126,22 @@ class ColorInputSwatch extends BaseColorInputAbstractSubview(BaseComponent(HTMLE
   }
 
   /**
-   The tabindex of the color preview.
+   The tabindex of the color preview button. 
+   So that we don't wind up with nested focusable elements,
+   the internal colorButton should should receive the tabIndex property, 
+   while the coral-colorinput-swatch should reflect the value using the _tabindex attribute.
    
    @type {Integer}
    @default 0
-   @htmlattribute tabindex
+   @htmlattribute _tabindex
    @htmlattributereflected
    */
   get tabIndex() {
```

Comment:
Does it make sense to keep the property ? Since we're doing a breaking change here in any case, I wonder if we shouldn't simple remove the property as well ?

Reply:
We can't eliminate the prop entirely, because `SelectableCollection` sets it, but we can remove the `tabIndex` attribute as soon as soon as it gets passed through to the `colorButton`.

Reply:
See: https://github.com/adobe/coral-spectrum/pull/67/commits/8ae6b3addc66f953bdaf22cd47a6df8e1b10e3e2